internal/cli/namespaces.go

```diff
@@ -47,6 +47,7 @@ func init() {
 
 	flags := CmdNamespaceDelete.Flags()
 	flags.BoolVarP(&force, "force", "f", false, "force namespace deletion")
+	flags.Bool("all", false, "delete all namespaces")
```

Comment:
I would try to define a "standard" for our flags. Personally I like the pointers because it will be already defined. You don't have to get the value with the `cmd.Flags().GetBool("all")` and check the error.

Reply:
The majority is actually not pointers (`Var` modifier) (The `P` modifier` allows for spec of a shorthand flag letter).
Hm. The `force` thing was added with 1844977e1515a59730f6542dddb01138e77c4349. Used `Var` and then later a regular `GetBool` instead of the variable.

Will see how that looks when it is correct.

Reply:
Done, commit 529aea3402fe8b101ae32f79858d6e0a50e607e9